<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>material_widget</class>
 <widget class="QWidget" name="material_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="4">
      <widget class="QPushButton" name="btn_shader">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="le_diffuse_plugin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_diffuse_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_11">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Plugin</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_shader_name">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mNormalLbl">
       <property name="text">
        <string>Normal Map</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mDiffuseLbl">
       <property name="text">
        <string>Diffuse Map</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="le_opacity_plugin">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btn_opacity">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="le_normal_plugin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btn_normal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Shader</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_normal_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="le_shader_plugin">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_opacity_name">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Opacity Map</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btn_diffuse">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="tb_norm_erase">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../toolkit.qrc">
         <normaloff>:/VecTB/icons/toolbars/Delete.png</normaloff>:/VecTB/icons/toolbars/Delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QToolButton" name="tb_opac_erase">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../toolkit.qrc">
         <normaloff>:/VecTB/icons/toolbars/Delete.png</normaloff>:/VecTB/icons/toolbars/Delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="tb_dif_erase">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../toolkit.qrc">
         <normaloff>:/VecTB/icons/toolbars/Delete.png</normaloff>:/VecTB/icons/toolbars/Delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QToolButton" name="tb_shader_erase">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../toolkit.qrc">
         <normaloff>:/VecTB/icons/toolbars/Delete.png</normaloff>:/VecTB/icons/toolbars/Delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Height Map</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_height_name"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="le_height_plugin"/>
     </item>
     <item row="5" column="3">
      <widget class="QToolButton" name="tb_height_erase">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../toolkit.qrc">
         <normaloff>:/VecTB/icons/toolbars/Delete.png</normaloff>:/VecTB/icons/toolbars/Delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="btn_height">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Material Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tb_material_color">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Specular Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tb_spec_color">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Specular Power</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sb_spec_power">
       <property name="maximum">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Shininess</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="sldr_mat_shininess">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Cull Mode</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cmb_cull_mode">
       <item>
        <property name="text">
         <string>Back Face</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Front Face</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Front and Back</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Culling</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="cb_cull_enable">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Color Only</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="cb_color_mode">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Wireframe</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="cb_wireframe">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Preview Mesh</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="cmb_preview_mesh">
       <item>
        <property name="text">
         <string>Plane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Half Tile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sphere</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cube</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../toolkit.qrc"/>
 </resources>
 <connections/>
</ui>
